MiniBootcamp/MiniBootcampTests/TweetModelTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  TweetModelTests.swift
+//  MiniBootcampTests
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import XCTest
+@testable import MiniBootcamp
+
+final class TweetModelTests: XCTestCase {
+    func test_mapToViewModel() {
```

Comment:
You should be consistent on the naming convention of your test methods. Either use this snake case or the camel case used on the other tests. Not big deal but consistency is great.